change/@ni-nimble-blazor-92675eea-194d-4c8d-81ea-ff1049cd4f6c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Gonna move it to draft. This change is in an unclear middle ground where it's behaving monolithic and like separate packages. The angular workspace isn't the ideal setup, I'd love for the angular tooling to be normal and exist like any other npm package instead of a custom workspace. Want to discuss it to make sure the responsibilities of the packages are clear if we go this way. Can discuss it post standup I guess since we didn't didcuss before the PR went to owners.

Reply:
points I want to discuss because I'll forget otherwise:
- the [eventual cdn packages](https://github.com/ni/nimble/pull/827/files#diff-0967e47790c3f683cc13aff0bdc76ecdcf42ebe20f0d9aadcb89b4fe7bf2ed5aR26) as owners of all-components
- long-term direction of actual separate packages, publishing an npm blazor package for efficient builds
- vs behavior of a monolithic workspace, top-level should be only thing participating, sub packages almost no deps or scripts

Reply:
This feels like a good structure to me, but we can discuss and see if any better solutions come up. Also, I haven't been getting buddy reviews on these Spright PRs, because I don't think others would have much to say about these changes. If you were expecting me to get buddies on these, just let me know.

Reply:
The concern for these in particular is that they are large changes that come as a surprise. Buddying with others not as familiar with the infrastructure isn't the concern. The concern is big infra changes not discussed at all until owner's PR.